Formula/minipro.rb

```diff
@@ -17,7 +17,7 @@ class Minipro < Formula
   depends_on "srecord"
 
   def install
-    system "make", "CC=clang"
+    system "make", "CC=clang", "PREFIX=#{prefix}"
```

Comment:
```suggestion
    system "make", "CC=#{ENV.cc}", "PREFIX=#{prefix}"
```
While we're making changes here, we may as well do this.

Reply:
done